### PR TITLE
Dates documentation draft

### PR DESCRIPTION
**Pull request by dipayan1985**

### CODE REVIEW

File: webapps/core/src/main/asciidoc/guides_dates.adoc [hunk 1]
```diff
@@ -18,17 +14,22 @@ The earliest date at which valid metadata has been provided by the submitter. Th
...
+=== `Last updated by submitter`
 
-The last date at which the sample was updated. Samples can be updated for curation needs and other technical purposes. More information about curation is available in the documentation [https://www.ebi.ac.uk/biosamples/docs/guides/curation] This attribute is generated by BioSamples.
+The last date at which the sample metadata was updated by the submitter.
 
-== Other dates in BioSamples
+=== `Last updated by curation`
 
-=== `Collection date`
+The last date at which a sample has been curated. Curation of samples is done daily by automated curation pipelines for technical pruposes or curation can be done manually by users as well.
```
Comment (mcourtot) [inline webapps/core/src/main/asciidoc/guides_dates.adoc:23]: Curation of samples is done periodically, and includes curation by automated pipelines or manually by users. Curation differs from submitter updates as the original record is not modified; changes are added as an overlay.

File: webapps/core/src/main/asciidoc/guides_dates.adoc [hunk 2]
```diff
@@ -6,10 +6,6 @@ BioSamples keeps records of different dates related to the sample lifecycle. The
 
 == Different dates 
 
-=== `ID Created on`
-
-The date at which the sample accession is created. This attribute is generated by BioSamples. IDs can be created in advance of collection or submission; BioSamples allows the pre-registration of sample accession to support cross-archive data exchange and data provenance management
-
```
Comment (mcourtot) [inline webapps/core/src/main/asciidoc/guides_dates.adoc:12]: We need to explain why this is not displayed anymore. can we say 
=== `ID Created on` (JSON only) 
for example?

Or should we add to the FAQ "why was the "ID created on" field removed?"
Many users reported confusion interpreting the "ID created on" field. To address this issue, we have removed the field from the UI. It has been retained in the JSON source for advanced users familiar with the BioSamples processing pipelines and documentation. "ID created on" reflects the date at which the sample accession is created. This attribute is generated by BioSamples. IDs can be created well in advance of collection or submission; BioSamples allows the pre-registration of sample accession to support cross-archive data exchange and data provenance management. IDs can be created by internal pipelines as unallocated placeholders pending user submission, sometimes months in advance of the actual sample collection.


Maybe the FAQ is better? Thoughts?

File: webapps/core/src/main/asciidoc/guides_dates.adoc [hunk 3]
```diff
@@ -18,17 +14,22 @@ The earliest date at which valid metadata has been provided by the submitter. Th
 
 The user-supplied date at which the sample metadata is made available publicly for the first time.
 
-=== `Updated on`
+=== `Last updated by submitter`
 
-The last date at which the sample was updated. Samples can be updated for curation needs and other technical purposes. More information about curation is available in the documentation [https://www.ebi.ac.uk/biosamples/docs/guides/curation] This attribute is generated by BioSamples.
+The last date at which the sample metadata was updated by the submitter.
```
Comment (mcourtot) [inline webapps/core/src/main/asciidoc/guides_dates.adoc:19]: The last date at which the sample metadata was updated by the submitter. Submitter updates differ from curation as the original record is modified to include the changes.

File: webapps/core/src/main/asciidoc/guides_dates.adoc (hunk 1 above)
Comment (dipayan1985) [reply webapps/core/src/main/asciidoc/guides_dates.adoc]: Addressed

File: webapps/core/src/main/asciidoc/guides_dates.adoc (hunk 2 above)
Comment (dipayan1985) [reply webapps/core/src/main/asciidoc/guides_dates.adoc:12]: FAQ seems to be the place place for this to go in. Addressed now.

File: webapps/core/src/main/asciidoc/guides_dates.adoc (hunk 3 above)
Comment (dipayan1985) [reply webapps/core/src/main/asciidoc/guides_dates.adoc]: Addressed.

**Pull request merged**